api/ssi-bridge/src/services/verification-service.ts

```diff
@@ -95,10 +95,8 @@ export class VerificationService {
 			throw new Error('no valid server identity to check the credential.');
 		}
 		const isVerifiedCredential = await this.ssiService.checkVerifiableCredential(vc);
-		const trustedRoots = await this.getTrustedRootIds();
 
-		const isTrustedIssuer = trustedRoots && trustedRoots.some((rootId) => rootId === vc.issuer);
-		const isVerified = isVerifiedCredential && isTrustedIssuer;
+		const isVerified = isVerifiedCredential && serverIdentity.id === vc.issuer;
```

Comment:
Correct me if I'm wrong but this would now always affect the check-credentials endpoints. So we would not rely anymore on the trusted roots when checking them. But this should constraint of the issuer needs to be the server identity shall only apply for the creation and revocation of an identity.